### DISCUSSION TITLE
Do you have any other interesting projects?

### DISCUSSION THREAD

**Discussion by Seven0492** [Uhuh/RoleBot#16] [General]
I mean, I looked and the one that picked my interest, GitGrader, and it seems interesting.

Have any other interesting projects?

**Comment by Seven0492** [comment]
And, huh, just noticed but CodeQL seems to be for Javascript, do what you will with this information.

**Comment by Uhuh** [comment]
>  I mean, I looked and the one that picked my interest, GitGrader, and it seems interesting.
GitGrader was a thing for university that could've been made into something cool. I moved it to Gitlab or something years ago for the class so I could have other team members join but it's pretty mute and stopped after the class finished.

It could've been something good but I wasn't that great of a webdev at the time. Knowing what I know now maybe it'd be possible and a lot easier.

Other interesting projects? No, at least not yet. Chalking up another bot idea that has to do with idle/rpg elements but it's currently just in planning with some friends at the moment. Any other "project" of mine would be websites but only public ones at the moment are my personal site and RoleBot's site since I'm working with someone to do one in private until it's announced.

> And, huh, just noticed but CodeQL seems to be for Javascript, do what you will with this information.
I noticed it said JavaScript but it claimed it scanned all the lines, lemmie check out the CI stuff. :)

**Reply by Uhuh** [reply]
The codeql looks like it says it checks BOTH JS and TS so I'm not too concerned about it. Am I wrong?

**Reply by Seven0492** [reply]
And well, when/if you release your other bot I'd be interested to be notified, maybe in here.

**Comment by Seven0492** [comment]
No, its okay, its just because when I hovered over the completed check it only said Javascript, so I made a guess.

GitGrader seems really interesting, I've been suggesting my teacher to get weekly projects going but he abandoned the idea after trying one or two times and its actually frustrating, its just a do whatever you want period. Maybe with GitHub projects, if my classmates can figure out how to use git, its pretty confusing at first, we can actually get a class where we get structured projects in teams or solo to submit, get the competition spirit going and actually learn, so I'm about a 100% self learned in about everything informatics related I learn. If we could get my teacher to try again, GitGrader or something similar could be the key to make it easy enough to be worthwhile.

So if I ever become better at programming in general, I'd be interested to contribute as this is a really interesting project, note that I don't know if any other similar projects exist.

**Reply by Uhuh** [reply]
GitGrader was based on something alumni of the university at the time made. I highly suggest looking into it instead.

My goal was to try and make it a gui friendly thing rather than cli based.

https://github.com/redkyn/assigner

**Reply by Seven0492** [reply]
Still interesting to make it friendly to the eyes.

Thanks for the link!
